Source/Urho3D/Resource/Image.cpp

```diff
@@ -965,6 +965,7 @@ bool Image::LoadColorLUT(Deserializer& source)
     if (components != 3)
     {
         URHO3D_LOGERROR("Invalid image format, can not load image");
+        FreeImageData(pixelDataIn);
```

Comment:
@weitjong, concerning this: while Coverity was down, I put a carefully coded [scope guard](https://ricab.github.io/scope_guard/) together. I would be happy to add to Urho3D. It is very useful in this sort of situation with early returns.

Reply:
Sounds good to me.

Reply:
alright, I opened a separate PR